HW04/Tatsiana_Zenkovich/spec/my_methods_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'my_methods.rb'
+
+describe '#my_each' do
+  let(:a) { Array.new([1, 2, 3, 4, 5]) }
```

Comment:
variable name 😞 Honestly, it's a good option for `subject`
```suggestion
  subject { Array.new([1, 2, 3, 4, 5]) }
```

Reply:
I replaced it, thank you!